AdminWebsite/AdminWebsite/ClientApp/src/app/services/justice-users.service.spec.ts

```diff
@@ -134,6 +134,25 @@ describe('JusticeUsersService', () => {
         });
     });
 
+    describe('clearUsers', () => {
+        it('should clear the search term', (done: DoneFn) => {
+            // arrange
+            clientApiSpy.getUserList.and.returnValue(of([]));
+
+            // act
+            service.clearUsers();
```

Comment:
Dont think you need this in the test, you clear on line 152 aswell

Reply:
you are right. I have removed it